packages/cli/src/utils/compiler.ts

```diff
@@ -38,3 +44,42 @@ export const compile = async (projectPath: string, entryFile: string, outputName
             },
         },
     });
+
+export const compilePlatformApp = async ({ outputName }: CompilerOptions) => {
+    const getHash = (text) => createHash('sha256').update(text).digest('hex');
+    const htmlHashPlugin = {
+        name: 'html-hash',
+        enforce: 'post',
+        transformIndexHtml(html, { bundle }) {
+            const linkTag = `${outputName}.${getHash(bundle['index.css'].source)}.css`;
+            const scriptTag = `${outputName}.${getHash(bundle['index.js'].code)}.js`;
```

Comment:
As these are not tags, perhaps `cssFileName` and `jsFileName` would be more descriptive